.changeset/gold-onions-press.md

```diff
@@ -0,0 +1,52 @@
+---
+"@salt-ds/lab": minor
+---
+
+- Refactored `Dialog` to use floating-ui and Salt's `Scrim`.
+- Implemented desktop support.
+- Added optional `closeOnBlur` prop to prevent a click away dismissing the dialog.
+- Added a `size` prop which takes `small`, `medium` and `large`.
+
+```tsx
+const AlertDialogTemplate = () => {
```

Comment:
This example should be updated to be more realistic right now it's of a template which seems weird.